packages/bundler-plugin-core/src/sentry/telemetry.ts

```diff
@@ -57,16 +56,22 @@ export function addSpanToTransaction(
   return span;
 }
 
-export function captureMinimalError(error: unknown | Error | AxiosError, hub: Hub) {
-  const isAxiosError = error instanceof AxiosError;
-  const sentryError =
-    error instanceof Error
-      ? {
-          name: `${isAxiosError && error.status ? error.status : ""}: ${error.name}`,
-          message: error.message,
-          stack: error.stack,
-        }
-      : {};
+export function captureMinimalError(error: unknown | Error, hub: Hub) {
+  let sentryError;
+
+  if (typeof error === "object") {
```

Comment:
Ah, probably should be `error && typeof error === 'object'`, as `typeof null === 'object` (😭 )

Reply:
Thanks, definitely would have missed that 😅